### ISSUE TITLE
Display a warning message in Genomic filters when panel lock is enabled

### ISSUE THREAD

**Issue by jmjuanes** [opencb/jsorolla#284]
## Description

If the panel lock option is enabled, after issue #274 all genomic filters will be intersected with the panel regions. We are displaying a warning message in the **Disease panels** filter:

![image](https://user-images.githubusercontent.com/5751201/144598419-da90ffc2-b173-401a-bd1a-8084ad48c188.png)

But no warning message has been added to the **Genomic** filters. Is important to also display a warning message to prevent future misunderstandings and issues.

## Proposed improvement

Display a warning message in the **Genomic** filters group when the panel lock option is enabled. We will need to extend the filter's configuration to allow displaying custom warning messages.

The final filters configuration will be the following:

```
{
    id: "consequence-type",
    title: "Select SO terms",
    visible: string || function,
    disabled: boolean || function,
    message: {
        visible: boolean || function,
        type: string,
        text: string,
    },
    description: string || function,
    tooltip: tooltips.consequenceTypeSelect,
    params: {
        consequenceTypes: this.consequenceTypes || CONSEQUENCE_TYPES,
    },
}
```

## Additional notes

This issue replaces the initial solution proposed in #255.

**Comment by jmjuanes** [comment]
Implemented custom message in filters (commit 207247) and display a warning message in region filters when panel lock is enabled (commit df07bd5):

![image](https://user-images.githubusercontent.com/5751201/145568988-4b7bcd13-1496-432b-890c-da23e9982a3a.png)

**Comment by jmjuanes** [comment]
@imedina This issue has been merged into `release-2.2.x` branch. Please could you review and improve the messages displayed in both filters? 

There is the list of messages to review:
- https://github.com/opencb/jsorolla/blob/release-2.2.x/src/webcomponents/variant/interpretation/variant-interpreter-browser-cancer.js#L441 
- https://github.com/opencb/jsorolla/blob/release-2.2.x/src/webcomponents/variant/interpretation/variant-interpreter-browser-cancer.js#L450
- https://github.com/opencb/jsorolla/blob/release-2.2.x/src/webcomponents/variant/interpretation/variant-interpreter-browser-rd.js#L473
- https://github.com/opencb/jsorolla/blob/release-2.2.x/src/webcomponents/variant/interpretation/variant-interpreter-browser-rd.js#L482

**Comment by pamag** [comment]
## Reviewed
commit: f104bd4712825f4c4fdc278c088a4b7b7bff1bf0

FIXED
